_episodes/02-generalsyntax.md

```diff
@@ -0,0 +1,264 @@
+---
+title: "General Syntax of a Shell Command"
+teaching: ??
+exercises: ??
+questions:
+- "How do I tell the shell to do something?"
+- "How do flags and options modify a shell program?"
+- "How do I find more information about specific shell commands?"
+objectives:
+- "Explain the correct general format of a shell command."
+- "Explain man pages and the --help flag."
+- "Explain how to supply options/flags to a shell command."
+keypoints:
+- "The basic format of a shell command is <command> <flags/options> <arguments>."
+- "More information about commands and their flags can be found in the manual, or 'man' pages."
+- "The --help option is also useful for learning more about commands."
+- "Some commands have default arguments."
+- "Man pages are also available on the internet."
+---
+## General syntax of a shell command
+Consider the command below as a general example of a command,
+which we will dissect into its component parts:
+
+~~~
+$ ls -F /
+~~~
+{: .language-bash}
+
+
+`ls` is the **command**, with an **option** `-F` and an
+**argument** `/`. Options (also called **switches** or **flags**)
+either start with a single dash (`-`) or two dashes (`--`), and they change the behaviour of a command.
+Arguments tell the command what to operate on (e.g. files and directories).
+Sometimes options and arguments are referred to as **parameters**.
+A command can be called with more than one option and more than one argument, but a
+command doesn't always require an argument or an option.
+
+Each part is separated by spaces: if you omit the space
+between `ls` and `-F` the shell will look for a command called `ls-F`, which
+doesn't exist. Also, capitalization can be important: `ls -r` is different from `ls -R`.
+
+Putting all that together, our command above gives us a listing
+of files and directories in the root directory `/`.
+An example of the output you might get from the above command is given below:
+
+~~~
+$ ls -F /
+Applications/         System/
+Library/              Users/
+Network/              Volumes/
+~~~
+{: .output}
+
+> ## Default Flags
+>
+> Some commands, when run without any flags, will assume one or more by default.
+> A good example of this is the 'wc' command, which has the default flags -c, -l,
```

Comment:
Backticks for `wc` please.

Some questions learners might have at this point on seeing this new command:
- What is the wordcount command?
- What does it do?
- Can I run an example of it?

Reply:
Thinking this over, to really test wc you'd need to make a file or have one handy to run wc on. Making files seems like a lot for this early lesson- what do you think a good way to deal with this would be?

Reply:
I would drop this section.  Not sure what others think?

Reply:
The impetus for creating the PR was better clarity around WC and its default flags but I'm not married to it being there. It doesn't seem like there's a good place to fit it in.

Reply:
I've taken it out, which should address both of the two comments above.